src/Epic/index.tsx

```diff
@@ -90,6 +92,8 @@ export const Epic: React.FC<EpicProps> = (props: EpicProps) => {
         <ThemeProvider theme={brand}>
             <div css={styles.epicWrapper}>
                 <section css={styles.epicContainer}>
+                    {children}
```

Comment:
Is there a better name than `children` here? Children is fine and technically accurate, but it doesn't convey the context around how this element is used. This has semantic significance here from where in the markup it is placed that we could convey with this name. Perhaps this is part of a wider interface that I haven't spotted, in which case the specific context here doesn't apply so we shouldn't pick a specific name? If not, perhaps mentioning that this appears as a header would be helpful?

Reply:
Good questions. The `children` prop is a [React thing](https://reactjs.org/docs/composition-vs-inheritance.html#containment), so if you specify your jsx like this:

```jsx
<A>
  <B />
</A>
```

Then the inner content of `A` is made available to `A` as `children`. But there are other options we could use here to give it a better name. We could capture the arg into a different name using destructuring like this:

```jsx
const A = ({ children: specialHeader ) => { ... }
```

Or alternatively pass the special header like any other prop, like this:

```jsx
<A specialHeader={<B />} />
```

(That last example is how you'd have to do things if you wanted to support a custom header _and_ footer for example).

Any thoughts on which of those might be preferable?

Reply:
It feels a bit different to the React docs there because as well as being "children" it is also **before** other hard-coded children elements (like the `<span>` and the paragraphs. No harm if this makes React sense though, very happy to defer to react expertise!

Reply:
I've update this, the prop in the `Epic` component is now called `headerSection`. So the new component implementation basically looks like this:

```typescript
export const EpicWithSpecialHeader: React.FC<EpicProps> = (props: EpicProps) => (
    <Epic {...props} headerSection={<HeaderSection />} />
);
```